### PR TITLE
convert: uses new par_map to work in parallel

### PR DESCRIPTION
**Pull request by vvvrrooomm**
## Description

Builds on  #3153 
parallelizes convert using par_map on python3

## To Do

- [ ] Documentation. (If you've add a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
- [ ] Changelog. (Add an entry to `docs/changelog.rst` near the top of the document.)
- [ ] Tests. (Encouraged but not strictly required.)

### CODE REVIEW

File: beetsplug/convert.py [hunk 1]
```diff
@@ -183,8 +185,8 @@ def commands(self):
 
     def auto_convert(self, config, task):
         if self.config['auto']:
-            for item in task.imported_items():
-                self.convert_on_import(config.lib, item)
+            items = zip(repeat(config.lib), list(task.imported_items()))
+            par_map(self.convert_on_import, items)
```
Comment (wisp3rwind) [inline beetsplug/convert.py:189]: Maybe make this `par_map(lambda item: self.convert_on_import(config.lib, item), task.imported_items())` with the old function signature of `convert_on_import`. In my opinion, that would be a lot clearer and should do precisely the same thing, right? I don't think there's any need to make a list out of `task.imported_items()`.

**Review by wisp3rwind — CHANGES_REQUESTED** [review]

File: beetsplug/convert.py (hunk 1 above)
Comment (vvvrrooomm) [reply beetsplug/convert.py]: done

**Review by wisp3rwind — CHANGES_REQUESTED** [review]
Looking good! Would you address the style issues and add a changelog entry? Then we can go ahead and merge.

**Review by wisp3rwind — APPROVED** [review]

**Comment by wisp3rwind** [comment]
I went for squashing the PR since the final changeset is so much easier to understand. I hope that's alright with you! Thanks again :tada:

**Pull request merged**